halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/plugins/Plugin.java

```diff
@@ -29,7 +29,7 @@
 
   private String id;
   private Boolean enabled;
-  private String uiResourceLocation;
+  @Deprecated private String uiResourceLocation;
```

Comment:
Left this here so people's configs don't blow up if they still have this value. I could be convinced it's better to blow up than silently not do anything.

Couldn't find a good pattern for deprecating / removing a parameter like this. There's the `@ValidForSpinnakerVersion` annotation, but this isn't tied to a Spinnaker version.